core/controllers/admin_test.py

```diff
@@ -38,26 +38,28 @@ def setUp(self):
     def test_admin_page_rights(self):
         """Test access rights to the admin page."""
 
-        response = self.testapp.get('/admin')
+        response = self.get_html('/admin', expect_errors=True,
```

Comment:
Here and elsewhere, status codes beginning with 2 or 3 should not be considered errors.

Reply:
If that's the case then should'nt the condition [here](https://github.com/oppia/oppia/blob/develop/core/tests/test_utils.py#L488) change to assert both 2xx and 3xx codes?

Reply:
done